chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -49,8 +56,12 @@ The ChainIndexer [periodically runs](https://github.com/filecoin-project/lotus/b
 1. **Archival Nodes**: **Keep GC disabled** (`GCRetentionEpochs` = 0) to retain all indexed data.
 
 2. **Non-Archival Nodes**:  Set `GCRetentionEpochs` to match the amount of chain state your node retains 
-(*Example:* if your node is configured to retain 2 days of Filecoin chain state with the Splitstore, set `GCRetentionEpochs` to `retentionDays * epochsPerDay = 2 * 2880 = 5760`).
-**Warning:** Setting this value below the chain state retention period will degrade RPC performance and reliability because the Index will lack data for epochs still present in the chain state.  
+
+**Example:** if your node is configured to retain Filecoin chain state with a Splitstore Hotstore that approximates 2 days of epochs, set `GCRetentionEpochs` to at least `retentionDays * epochsPerDay = 2 * 2880 = 5760`).
+
+**Warning:** Setting this value below the chain state retention period will degrade RPC performance and reliability because the ChainIndexer will lack data for epochs still present in the chain state.
```

Comment:
```suggestion
**Warning:** Setting this value below the chain state retention period may degrade RPC performance and reliability because the ChainIndexer will lack data for epochs still present in the chain state.
```

in the glif case, users can't request data past a certain lookback, which I believe they set to 2880 epochs, so that's where it matters for them, they may use splitstore, but are unlikely to get requests beyond 2880 anyway so who really cares?